metrics/config.go

```diff
@@ -201,8 +206,8 @@ func getMetricsConfig(ops ExporterOptions, logger *zap.SugaredLogger) (*metricsC
 
 // UpdateExporterFromConfigMap returns a helper func that can be used to update the exporter
 // when a config map is updated.
-// DEPRECATED. Use UpdateExporter instead.
-func UpdateExporterFromConfigMap(domain string, component string, logger *zap.SugaredLogger) func(configMap *corev1.ConfigMap) {
+func UpdateExporterFromConfigMap(component string, logger *zap.SugaredLogger) func(configMap *corev1.ConfigMap) {
```

Comment:
Do you intend to use this function in serving repo? I didn't find this is being used in https://github.com/knative/serving/pull/4163.

Reply:
It is, but indirectly.  This enables the entire controller `main()` to be defined in `knative/pkg` and simply called from Serving (or Eventing).

Reply:
At whoops, not sure if the PR you linked does it quite yet, but https://github.com/knative/pkg/pull/423 and https://github.com/knative/serving/pull/4162 do what I mentioned.